### PR TITLE
Convert to Grant

### PR DESCRIPTION
**Pull request by aamustapha**
##### Description
Allow Hackathon team members to spin off a grant from an existing hackathon.

##### Refers/Fixes
https://github.com/gitcoinco/web/issues/7098
https://github.com/gitcoinco/web/pull/7359

##### Testing

https://www.loom.com/share/a833c6e9bae54c70a5a6617ef7d72799

### CODE REVIEW

**Comment by thelostone-mc** [comment]
- Could you throw in a video ? 
- Make sure matic grant creation flow works

**Review by thelostone-mc — APPROVED** [review]

**Comment by thelostone-mc** [comment]
@aamustapha what do you think about hiding the convert to grant button once the project has been made into a grant ! It doesn't make sense to have that button there once it's bren converted right ? ( Noticed this on the video )!

**Comment by aamustapha** [comment]
Yes, I have done that in the updates

File: app/dashboard/templates/project/detail/info.html [hunk 1]
```diff
@@ -50,6 +50,9 @@
           {% endif %}
 
           {% if is_member %}
+            {% if not project_obj.grant_url %}
+              <a :href="`/grants/new?title=${project.name}&description_rich=${project.summary}&reference_url=${project.work_url}&github_project_url=${project.work_url}&related_hackathon_project_id=${project.id}`" target="_blank" class="project__action--admin btn button-white btn-block font-subheader btn-lg m-2"><i class="fa fa-plus mr-2"></i>CONVERT TO GRANT</a>
+            {% endif %}
```
Comment (aamustapha) [inline app/dashboard/templates/project/detail/info.html:55]: Here @thelostone-mc.

**Comment by thelostone-mc** [comment]
tested to make sure grant creation works! works chill

**Pull request merged**